### ISSUE TITLE
JSON format results in no output

### ISSUE THREAD

**Issue by mattroberts297** [piotrmurach/github_cli#36]
Hey thanks for making this CLI. I'm experiencing an issue with the JSON format.

This works:
```
gcli config core.format table
gcli repo list
```
Result:
```
Response Status: 200
┏━━━━━━━━━━━┳━━━━━━━━━━━━┓
... <snip>
```

This does not work:

```
gcli config core.format json
gcli repo list
```

Result:
```
Response Status: 200

```

I tried installing `gem install json` per another issue, but no luck.

I am on mac os x with the following gems:
```
gem list --local

*** LOCAL GEMS ***

addressable (2.5.2)
bigdecimal (default: 1.3.4)
bundler (default: 1.16.0)
cmath (default: 1.0.0)
csv (default: 1.0.0)
date (default: 1.0.0)
dbm (default: 1.0.0)
descendants_tracker (0.0.4)
did_you_mean (1.2.0)
etc (default: 1.0.0)
faraday (0.12.2)
fcntl (default: 1.0.0)
fiddle (default: 1.0.0)
fileutils (default: 1.0.2)
github_api (0.18.2)
github_cli (0.6.2)
hashie (3.5.7)
io-console (default: 0.4.6)
ipaddr (default: 1.2.0)
json (default: 2.1.0) # also tried json-1.8.6
jwt (1.5.6)
minitest (5.10.3)
multi_json (1.13.1)
multi_xml (0.6.0)
multipart-post (2.0.0)
net-telnet (0.1.1)
oauth2 (1.4.0)
openssl (default: 2.1.0)
power_assert (1.1.1)
psych (default: 3.0.2)
public_suffix (3.0.2)
rack (2.0.4)
rake (12.3.0)
rdoc (default: 6.0.1)
scanf (default: 1.0.0)
sdbm (default: 1.0.0)
stringio (default: 0.0.1)
strscan (default: 1.0.0)
test-unit (3.2.7)
thread_safe (0.3.6)
tty (0.0.11)
webrick (default: 1.4.2)
xmlrpc (0.3.0)
zlib (default: 1.0.0)
```

**Comment by jflo-elsevier** [comment]
This also fails to work with command line args:

`gcli repo list --format=json --no-pager`

**Comment by piotrmurach** [comment]
Fixed and will be part of `0.7.0` release.
